node/src/Pos/Delegation/Cede/Holders.hs

```diff
@@ -59,12 +60,18 @@ evalMapCede = flip Ether.evalLazyStateT
 
 instance MonadDBRead m => MonadCedeRead (MapCede m) where
```

Comment:
I've already asked why constraint isn't `MonadCedeRead m`, IIRC you agreed it was better to use `MonadCedeRead`

Reply:
Can't/don't remember me saying that. I either implement what you say and do not implement `MonadDBRead (MayCede m)` constraint or vice versa. I chose option 1. They are more or less the same imho.